package.json

```diff
@@ -62,9 +62,11 @@
     "@typescript-eslint/eslint-plugin": "^4.28.3",
     "@typescript-eslint/parser": "^4.28.3",
     "chai": "^4.3.4",
+    "codecov": "^3.8.3",
```

Comment:
Is this really needed? Is it that deprecated uploader? It won't be used for uploading, the action has its own uploader (the new one).

Reply:
Right, with action we don't really need it.